Sources/PackageDescription/PackageDependency.swift

```diff
@@ -20,24 +20,43 @@ extension Package {
     /// A package dependency consists of a Git URL to the source of the package,
     /// and a requirement for the version of the package.
     ///
-    /// The Swift Package Manager performs a process called *dependency resolution* to
-    /// figure out the exact version of the package dependencies that an app or other
-    /// Swift package can use. The `Package.resolved` file records the results of the
-    /// dependency resolution and lives in the top-level directory of a Swift package.
-    /// If you add the Swift package as a package dependency to an app for an Apple platform,
-    /// you can find the `Package.resolved` file inside your `.xcodeproj` or `.xcworkspace`.
+    /// Swift Package Manager performs a process called _dependency resolution_ to figure out
+    /// the exact version of the package dependencies that an app or other Swift
+    /// package can use. The `Package.resolved` file records the results of the
+    /// dependency resolution and lives in the top-level directory of a Swift
+    /// package. If you add the Swift package as a package dependency to an app
+    /// for an Apple platform, you can find the `Package.resolved` file inside
+    /// your `.xcodeproj` or `.xcworkspace`.
     public class Dependency: Encodable {
+        /// The kind of dependency.
         @available(_PackageDescription, introduced: 5.6)
         public enum Kind {
+            /// A dependency located at the given path.
+            /// - Parameters:
+            ///    - name: The name of the dependency.
+            ///    - path: The path to the dependency.
             case fileSystem(name: String?, path: String)
+            /// A dependency based on a source control requirement.
+            ///  - Parameters:
+            ///    - name: The name of the dependency.
+            ///    - location: The Git URL of the dependency.
+            ///    - requirement: A `SourceControlRequirement` instance.
```

Comment:
Should this also be `The version based requirement for a package` like the case below?  It's exactly the same use, and when seen in the code completion or help in Xcode, it's already apparent that the type is `SourceControlRequirement`.